### ISSUE TITLE
[ticket] Add solo ticket purchase

### ISSUE THREAD

**Issue by vctt94** [decred/decrediton#2086]
We will probably need to discuss this better as to vote the wallet will need to be unlocked, so I believe having a config option with some kind of warning would be good.

**Comment by jrick** [comment]
This should only be used to buy tickets for a separate voting wallet with a different seed.  It can prevent the user from continuing if the voting address is controlled by decrediton.

**Comment by vctt94** [comment]
That makes sense

**Comment by matheusd** [comment]
We also need to add all sorts of warnings/confirmations, since using some random address (not controlled by the wallet) for voting rights implies that

- You're ceding consensus voting rights to (potentially) someone else
- If the owner of this address "disappears", you might get locked out of your funds
- All solo-voting issues apply (have multiple online, unlocked wallets, etc etc)

Given that a sizeable % of decrediton users haven't really internalized staking dynamics yet (eg: see how many people complain about "locked tickets", "tickets haven't voted after a month", etc) IMO solo voting needs to be something that takes some effort to enable.

**Comment by vctt94** [comment]
Yeah, I agree with matheusd on this one, as a use can get deceived, thinking they will not pay fee or something else and ending with a missed ticket.

We should think in a way to make sure ppl who are not supposed to use it, use it
